.github/github_token_rotation.md

```
@@ -0,0 +1,399 @@
+     } | jq -e 'length > 0' >/dev/null && \
+     echo "Secret retrieved successfully and validated"
+```
+
+- **Expected Output**: The current GitHub token value stored in AWS Secrets Manager.
+- **Validation**: Ensure the response contains a valid token string, which indicates that Secrets Manager is correctly storing and providing access to the token.
+
+- **GitHub API Debugging**
+
+  To confirm that the GitHub App is authorized and properly configured, use this command to retrieve the app details:
+
+# Verify GitHub App configuration
```

Markdownlint

Multiple top-level headings in the same document

(MD025, single-title, single-h1)